### DISCUSSION TITLE
How do I disable math rendering in Quarto when using Hugo?

### DISCUSSION THREAD

**Discussion by disidrosi** [quarto-dev/quarto-cli#10702] [Q&A]
### Description

I am rendering `.md` files for a Hugo website using Quarto. I have katex rendering already configured on my Hugo website, but this conflicts with the default math rendering from Quarto.

Specifically, when I call quarto preview on a `.qmd` file included within my Hugo project, I get a js injection within the resulting `.md` file, which prevents Hugo from displaying math correctly. The js I'm talking about is the following:

```javascript
<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js" integrity="sha512-c3Nl8+7g4LMSTdrm621y7kf9v3SDPnhxLNhcjFJbKECVnmZHTdo+IRO05sNLTH/D3vA6u1X32ehoLC7WFVdheg==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous" data-relocate-top="true"></script>
<script type="application/javascript">define('jquery', [],function() {return window.jQuery;})</script>
```

If I remove these lines manually, everything works correctly. Obviously I'd like to control the behavior from the frontmatter of my `.qmd` file, and avoid the javacript injection altogether, but I can't seem to figure out how to do that correctly.

For reference, the frontmatter I am using is:

```yaml
---
title: Some title here
date: '2024-08-31'
format: hugo-md
jupyter: python3
math: true
---
```

I tried adding `tex_math_dollars: false` to my frontmatter, thinking that this would control the underlying pandoc's behavior directly, but it doesn't work. Can anyone give me a suggestion on what I am doing wrong?

For reference, I am using Quarto 1.5.55 and Hugo v0.134.0+extended, rendering from VSCode using the official Quarto extension.

**Comment by mcanouil** [comment]
Could you share a **small self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? The goal is to make it as easy as possible for us to recreate your problem so that we can fix it: please help us help you! Thanks.

This being said, I would say to set `html-math-method: plain`.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.
